android/build.gradle

```diff
@@ -46,7 +46,7 @@ android {
 }
 
 dependencies {
-    implementation 'com.facebook.react:react-native:0.59.10'
+    implementation 'com.facebook.react:react-native:0.61.5'
```

Comment:
Are we changing to this version in sdk ?

Reply:
we are indeed

Reply:
it actually already was, this library was just outdated. Soon we will come back to update this to 64